Project/SubZeroDLL/dllmain.cpp

```diff
@@ -0,0 +1,117 @@
+#include "pch.h"
+#include "HttpClient.h"
+#include "ReflectiveLibraryLoader.h"
+#include "../SubZeroDriver/SubZeroCommon.h"
+#include "../SubZeroClient/SubZeroLoaderCommon.h"
+#include "../SubZeroUtils/RegistryManager.h"
+#include "../SubZeroUtils/ServiceManager.h"
+#include "../SubZeroUtils/AutoRegistryKeyHandle.h"
+
+const std::string IpAddress = "192.168.14.139";
```

Comment:
Decide on your coding conventions:
In my opinion `const` should be in UPPER_CASES.
What ever you decided keep on unity all over your project.